chore: align README for non-prefixed packages (#2994)
vaadin/web-components: packages/virtual-list/README.md

# @vaadin/virtual-list

> A web component allowing to render a long list of items without sacrificing performance.

Reviewer: ```suggestion
A web component for rendering a long list of items without sacrificing performance.
```

Author: Done.
